man/repart.d.xml

```diff
@@ -327,7 +327,71 @@
         data is never overwritten. Note that the data is copied in before the partition table is updated,
         i.e. before the partition actually is persistently created. This provides robustness: it is
         guaranteed that the partition either doesn't exist or exists fully populated; it is not possible that
-        the partition exists but is not or only partially populated.</para></listitem>
+        the partition exists but is not or only partially populated.</para>
+
+        <para>This option cannot be combined with <varname>Format=</varname> or
+        <varname>CopyFiles=</varname>.</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>Format=</varname></term>
+
+        <listitem><para>Takes a file system name, such as <literal>ext4</literal>, <literal>btrfs</literal>,
+        <literal>xfs</literal> or <literal>vfat</literal>, or the special value <literal>swap</literal>. If
+        specified and the partition is newly created it is formatted with the specified file system (or as
+        swap device). The file system UUID and label are automatically derived from the partition UUID and
+        label. If this option is used, the size allocation algorithm is slightly altered: the partition is
```

Comment:
If the file system UUID is derived from the partition UUID, which would be a small pool considering the discoverable partition scheme, and the label, which would usually be something guessable, wouldn't there be a high chance for file system UUIDs coming out the same? I vaguely remember this being a potential issue for btrfs when they two file systems with the same UUID are mounted on the same machine. Could this be a problem?
I'm sorry if this is a silly question.

Reply:
GPT manages two UUIDs per partition. The partition UUID and the partition type UUID. The former is supposed to identify the partition uniquely world-wide, and is what we derive from the seed that is either the machine ID or random or whatever you specify. The latter identifies the type of the partition and is what is defined in the discoverable partition spec.

the gpt partition uuid should hence be as unique as that seed value you specify, and the since the fs uuid is derived from the partition uuid, the same is true for that.

Does that make sense?

Reply:
That makes sense. Thank you for the explanation!